### ISSUE TITLE
SSL error: cannot create a client socket with a PROTOCOL_TLS_SERVER context

### ISSUE THREAD

**Issue by vedang122** [opencybersecurityalliance/stix-shifter#1677]
**Describe the bug**
Encountered following error while using `transmission.ping()` for `okta` and `azure_log_analytics`(only confirmed for these two):
```
Cannot connect to host trial-4512277.okta.com:443 ssl:<ssl.SSLContext object at 0x10b003d10> [Cannot create a client socket with a PROTOCOL_TLS_SERVER context (_ssl.c:795)
```

**To Reproduce**
1. I'm using python 3.11
2. Install stix-shifter, stix-shifter-utils and stix-shifter-modules-okta using pip(default pip version given with python version download) in an environment 
3. Run following piece of code:
```
from stix_shifter.stix_transmission import stix_transmission
transmission = stix_transmission.StixTransmission(
       "okta", connection, config) 
transmission.ping()
```

**Expected behavior**
I expect ping to be successful. 

**Screenshots**
N/A

**Desktop (please complete the following information):**
 - OS: [e.g. iOS] MacOS 13.6.6
 - Browser [e.g. chrome, safari] Chrome
 - Version [e.g. 22] Python 3.11

**Additional context**
N/A

**Comment by pcoccoli** [comment]
Is this a duplicate of #1633?

**Comment by vedang122** [comment]
Looking at the context it seems so, but it's still not fixed for 3.11 in that case.

**Comment by pcoccoli** [comment]
You're right; it's fixed but not in a release yet. @mdazam1942 any idea when the next release will be?
